docs/node/run-your-node/keymanager-node/README.md

```diff
@@ -49,60 +49,52 @@ The key manager ParaTime requires the use of a TEE. See the [Set up trusted exec
 In order to configure the node create the `/node/etc/config.yml` file with the following content:
 
 ```yaml
-datadir: /node/data
+mode: keymanager
 
-log:
-  level:
-    default: info
-    tendermint: info
-    tendermint/context: error
-  format: JSON
+common:
+  data_dir: /node/data
+  log:
+    format: JSON
+    level:
+      default: debug
+      cometbft: info
+      cometbft/context: error
```

Comment:
```suggestion
      cometbft: info
      cometbft/context: error
      default: info
```
Please check other docs and keep the logging configuration and indentation same.